es/utils/crypto.js

```diff
@@ -359,18 +359,18 @@ export function prepareTx (signature, data) {
 }
 
 export function personalMessageToBinary (message) {
-  const p = Buffer.from('æternity Signed Message:\n', 'utf8')
+  const p = Buffer.from('aeternity Signed Message:\n', 'utf8')
```

Comment:
what is the problem with æ? As I know, we are using it on Ledger

Reply:
Hans suggested we use “ae” instead as otherwise we can face problems in the future.